qiskit/tools/visualization/_text.py

```diff
@@ -403,11 +410,14 @@ def fillup_layer(names):  # pylint: disable=arguments-differ
 class TextDrawing():
     """ The text drawing"""
 
-    def __init__(self, qregs, cregs, instructions, plotbarriers=True, line_length=None):
+    def __init__(self, qregs, cregs, instructions, circuit, plotbarriers=True,
+                 line_length=None):
         self.qregs = qregs
         self.cregs = cregs
         self.instructions = instructions
 
+        self.circuit = circuit
```

Comment:
is this needed? I could not find the usage...

Reply:
No sorry! Have removed it now.